docs/sources/developers/plugins/migration-guide/v9.x-v10.x/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Guide for migrating plugins from Grafana v9.x to v10.x
+keywords:
+  - grafana
+  - plugins
+  - migration
+  - plugin
+  - documentation
+title: Migrating plugins from Grafana 9.x to 10.x
+menutitle: v9.x to v10.x
+weight: 1900
+---
+
+# Migrating plugins from Grafana version 9.x to 10.x
+
+## Verifying plugin behaviour with React 18
+
+One of the major changes in Grafana 10 is our upgrade to React 18 and use of the new React client rendering API. These changes were delivered to the core `grafana` repo with [PR 64428](https://github.com/grafana/grafana/pull/64428).
```

Comment:
Given this is a doc about the major changes in Grafana 10 we can probably simplify this, suggestion: "Grafana 10 includes our upgrade to React 18..."